Assets/Scripts/dragAndDrop.cs

```diff
@@ -5,7 +5,6 @@
 public class dragAndDrop : MonoBehaviour
 {
     // Start is called before the first frame update
-    static bool canDrag = true;
```

Comment:
this is needed cause of static nature. To not double pick up as the canDrag variable is global